diesel_cli/src/database.rs

```diff
@@ -255,7 +255,7 @@ table! {
 fn pg_database_exists(conn: &PgConnection, database_name: &str) -> QueryResult<bool> {
     use self::pg_database::dsl::*;
 
-    pg_database.select(datname) // here come dat name!!!! o shit waddup!!!!
+    pg_database.select(datname)
```

Comment:
Not sure what this comment brings to the table, so I removed it.

Reply:
Was probably one of those 2017 jokes that no-one laughs about anymore.

But hey, it was introduced  #666!

Reply:
Drunk Sean thought it was funny